src/ui/components/InputRequest/InputRequest.tsx

```diff
@@ -67,19 +63,7 @@ const InputRequest = () => {
 
   const resolveConnectionOobi = async (content: string) => {
     // Adding a pending connection item to the UI.
-    // This will be removed when the create connection process ends.
-    const connectionName = new URL(content).searchParams.get("name");
-
-    const pendingId = randomSalt();
-    dispatch(
-      updateOrAddConnectionCache({
-        id: pendingId,
-        label: connectionName || pendingId,
-        status: ConnectionStatus.PENDING,
-        createdAtUTC: new Date().toString(),
-      })
-    );
-
+    // This will be removed when the create connection process ends.å
```

Comment:
Remove extra character at the end of the comment.

Reply:
Done